service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDetailsHelper.java

```diff
@@ -64,6 +69,22 @@ public static CaseDetails removeTemporaryFields(CaseDetails caseDetails, Class c
         return caseDetails;
     }
 
+    public static CaseDetailsMap addFields(CaseDetailsMap caseDetails, Object target, String... groups) {
```

Comment:
I'd say the second parameter is not the target (as in it's not the object that will be modified). It's actually the "model" (the object to be copied - or partially copied)